design/overview.md

```diff
@@ -81,43 +82,36 @@ And provide a security equivalent to or greater than IBC.
 
 ## Bootstraping
 
-This is based on [Installing Peggy on a live cosmos chain](notes.md#installing-peggy-on-a-live-cosmos-chain).
-We also assume the act of upgrading the Cosmos-based binary to have peggy module is already complete,
-as approaches to that are discussed in many other places. Here we focus on the *activation* step.
+We assume the act of upgrading the Cosmos-based binary to have peggy module is already complete,
+as approaches to that are discussed in many other places. Here we focus on the _activation_ step.
 
-Set up and register orchestrators:
+1. Each `Operator` generates an Ethereum and Cosmos private key for the `Orchestrator`. These addresses are signed and submitted by the Operators valoper key. The `Orchestrator` is now free to use these delegated keys for all Peggy messages.
+1. Each `Operator` configures their `Orchestrator` with the `Peggy Bridge ID` and `start threshold` they wish, their `Orchestrator` is to reject any proposals that do not agree on these values.
+1. Anyone submits a Peggy contract using a known codehash and the current validator set of the Cosmos zone to an Ethereum compatible blockchain. Note that step #1 must happen first because the `Orchestrator` Ethereum Addresses must be made known in order for verification to be possible.
+1. Each `Operator` may or may not configure their `Orchestrator` with the above Peggy contract address
+1. If configured with an address the `Orchestrator` checks the provided address. If the contract passes validation the `Orchestrator` signs and submits the contract address. Validation is defined as finding the expected code hash and a validator set matching the current set within `Allowed validator set delta`. This message also includes the proposed `Peggy Bridge ID` and `start threshold`
```

Comment:
> the `Orchestrator` signs and submits the contract address

Please define a name of a message here - so we can connect this to the code easily. No need to even spec out the struct, just the message names, so the flow connects to the types